<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password</class>
 <widget class="QDialog" name="password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(148,169,216);</string>
  </property>
  <widget class="QPushButton" name="pbPassword1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Press your Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLogin">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPassword8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; color:rgb(40,40,40); font-weight:bold;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
